### ISSUE TITLE
missing elevator music

### ISSUE THREAD

**Issue by nstringham** [SkyCryptWebsite/SkyCrypt#102]
**Describe the bug**
the console logs an error that it failed load https://sky.shiiyu.moe/misc/music/ding.mp3 and https://sky.shiiyu.moe/misc/music/elevator.mp3

**Profile/Location**
Any profile I eg. https://sky.shiiyu.moe/stats/Cookie_Wookie_7/Tomato

**Expected behavior**
The console should be clear of errors

**Screenshots**
![image](https://user-images.githubusercontent.com/44071655/95409076-e6105200-08ee-11eb-98ac-185802e562bb.png)

**Environments**
- Windows 10 Home 2004
  - Google Chrome
    - Version 86.0.4240.75
  - Firefox Browser
    - Version 81.0.1

- Device: OnePlus 6 (ONEPLUS A6003)
   OS: OxygenOS 10.3.5 based on Android 10
    - Google Chrome
      - Version 85.0.4183.127

**Likely Cause**
the non-exsistant files are referenced by [/views/stats.ejs:2353-2356](https://github.com/SkyCryptWebsite/SkyCrypt/blob/master/views/stats.ejs#L2353-L2356) wich are
```js
new Elevator({
    mainAudio: '/misc/music/elevator.mp3',
    endAudio: '/misc/music/ding.mp3'
});
```

**Comment by metalcupcake5** [comment]
Warp added this and never finished it. ¯\_(ツ)_/¯

**Comment by WarpWing** [comment]
Truly a shame <_<

**Comment by nstringham** [comment]
fixed by 8907d92339ac9a4eca8e5518bed637a892c27fbe
